src/nginxconfig/i18n/ru/templates/domain_sections/routing.js

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2020 DigitalOcean
+
+This code is licensed under the MIT License.
+You may obtain a copy of the License at
+https://github.com/digitalocean/nginxconfig.io/blob/master/LICENSE or https://mit-license.org/
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions :
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+import common from '../../common';
+
+export default {
+    fallbackRouting: 'Fallback маршрутизация',
```

Comment:
Hi! 👋
I think better use native programming word straight on English Language: all developers know "Fallback" term, because modern Russian Language full of "Original" IT words...
I can translate Fallback as `"Резервный"` (reserved, backup), but it can confuse some developers.
Also I can translate as `"Фолбек"` - this is a complete transcription that can understand any ru-lang speaker, but I don't thing that it's a good practice (non-literary).
What would you recommend?

Reply:
I definitely defer to whatever you think is best as the Russian speaker here, so if you think leaving it as `Fallback` is clearest, let's go with that!